src/gui/addConfigurationDialogFuncs.cpp

```diff
@@ -179,6 +180,10 @@ void AddConfigurationDialog::finalise()
     // Create the Configuration and a suitable generator
     auto *newConfiguration = dissolve_.addConfiguration();
     auto &generator = newConfiguration->generator();
+
+    // Add Temperature node, by default
+    generator.createRootNode<TemperatureProcedureNode>("Temperature");
```

Comment:
I'd just default this one so it gets the standard node naming
```suggestion
    generator.createRootNode<TemperatureProcedureNode>({});
```